components/docs-chef-io/content/automate/ha_aws_deploy_steps.md

```diff
@@ -25,9 +25,9 @@ Follow the steps below to deploy Chef Automate High Availability (HA) on AWS (Am
 - If you want to use Default VPC, we have to create public and private subnets, If subnets are unavailable. Please refer [this](https://docs.aws.amazon.com/vpc/latest/userguide/default-vpc.html)
 - We need three private and three public subnets in a vpc (1 subnet for each AZ). As of now, we support a dedicated subnet for each AZ.
 - We recommend creating a new vpc. And Bastion should be in the same VPC.
-- Get AWS credentials (`aws_access_key_id` and `aws_secret_access_key`) with privileges like: `AmazonS3FullAccess`, `AdministratorAccess`.
+- Attach IAM role to the Bastion with `AmazonS3FullAccess`, `AdministratorAccess` privileges or get AWS user credentials with the same privileges. Click [here](/automate/ha_iam_user/) to learn more about creating IAM Users.
 
-    Set these in `~/.aws/credentials` in Bastion Host:
+Set the AWS user credentials in `~/.aws/credentials` in Bastion Host:
 
     ```bash
     sudo su -
```

Comment:
why `sudo su -`  is needed ?

Reply:
As we are executing chef-automate with sudo permission, terraform will try to access credentials from /root folder. And only root user will have permission to write in that folder. So we need that